src/components/Navigation/Navigation.scss

```diff
@@ -166,7 +166,7 @@ $disabled-fade: 0.6;
       --p-none,
       linear-gradient(color(state, hover), color(state, hover))
     );
-    background-color: var(--p-interactive-neutral-elevation-2);
+    background-color: var(--p-neutral-action);
```

Comment:
I think these should be surface-based roles, more similar to action list.

Reply:
Makes sense to me.